src/fsharp/TypedTreeOps.fs

```diff
@@ -2796,7 +2802,8 @@ type DisplayEnv =
         printVerboseSignatures = false
         g = tcGlobals
         contextAccessibility = taccessPublic
-        generatedValueLayout = (fun _ -> None) }
+        generatedValueLayout = (fun _ -> None)
+        genericParameterStyle = GenericParameterStyle.Implicit }
```

Comment:
Is this the correct default?

Reply:
This preserves the behavior currently, which is to print based on the state of the flag on the TyCon.

I could see a future FCS release where we say 'nah let's just go for Prefix by default' and we'd change it here, though.